sympy/geometry/tests/test_line.py

```diff
@@ -286,6 +286,15 @@ def test_contains():
         assert len(w) == 1
 
 
+def test_coordinate():
+    p1, p2, p3 = Point(0, 1), Point(2, 3), Point(3, 4)
+    l = Line(p1, p2)
+    # Line parameter
+    assert l.coordinate(p3) == Rational(3, 2)
```

Comment:
also, `assert l.coordinate(p3.args) == Rational(3, 2)`

Reply:
Hey,
This line of code is giving a `Value Error: Other should be a point . `